tklbam/index.cgi

```diff
@@ -78,32 +78,45 @@ print ui_form_start('restore.cgi', 'post');
 printf "<div style='text-align: right; padding-right: 5px'><a href='list_refresh.cgi'>%s</a></div>", text('index_list_refresh');
 
 my @hbrs = tklbam_list();
+my @this_hbr = undef;
```

Comment:
afaik defined values have a value of `undef` if not specified so you could just do:

```perl
my @this_hbr;
```

Reply:
Awesome. Well spotted! I'll fix this and merge.